lib/bits_service_client/client.rb

```diff
@@ -112,22 +112,39 @@ def blob(key)
       )
     end
 
+    def signed_url(key, verb: nil)
```

Comment:
This got here by accident when we rebased. Please remove it. It lives in blob.rb now.

Reply:
this is gone with 
https://github.com/cloudfoundry-incubator/bits-service-client/pull/3/commits/ca5f4272f48959af6d20941a0441c214ba7a2df2